tests/nvmeof/test_ceph_nvmeof_neg_tests.py

```diff
@@ -1163,6 +1168,239 @@ def test_ceph_83581755(ceph_cluster, rbd, pool, config):
         return 0
 
 
+def test_ceph_83608266(ceph_cluster, rbd, pool, config, placement_cfg):
+    """CEPH-83608266: Deploy same GW node in different GWgroup.
+    Returns:
+        int: 0 on success, 1 on failure.
+    """
+    admin_node = ceph_cluster.get_nodes(role="installer")[0]
+    conf = {}
+    executor = ThreadPoolExecutor()
+    io_tasks = []
+    clients = []
+
+    def string_to_dict(string):
+        """Parse ANA states from the string."""
+        states = string.replace(" ", "").split(",")
+        dict = {}
+        for state in states:
+            if not state:
+                continue
+            _id, _state = state.split(":")
+            dict[int(_id)] = _state
+        return dict
+
+    def deploy_nvme_service(conf, gw_node, nvmegwcli, redeploy=False):
```

Comment:
this is available in the ha..py, let's use those attributes.

Reply:
The function name for deploying the NVMe service is the same as in `ha.py`, but the purpose is different. I’ve created a new version of the function that supports three types of deployments: by label, by nodes, and by spec.